cmd/snap-confine/mount-support.c

```diff
@@ -322,16 +322,32 @@ static void sc_bootstrap_mount_namespace(const struct sc_mount_config *config)
 		//  - https://bugs.launchpad.net/snap-confine/+bug/1580018
 		//  - https://bugzilla.opensuse.org/show_bug.cgi?id=1028568
 		const char *dirs_from_core[] = {
-			"/etc/alternatives", "/etc/ssl", "/etc/nsswitch.conf",
+			"/etc/alternatives", "/etc/nsswitch.conf",
 			// Some specifc and privileged interfaces (e.g docker-support) give
```

Comment:
not your typo, but if you need to push another change to this pr:

```suggestion
			// Some specific and privileged interfaces (e.g docker-support) give
```